GUIDELINES.md

```diff
@@ -171,6 +171,28 @@ Example:
 }
 ```
 
+### Request Nullable/Optional properties
+
+#### When to use the `null` in properties
+
+1. When you want to unassign the value of that property.
```

Comment:
This should probably state that this is during `update` calls?

Reply:
We can make it more explicit yes. But I assumed that to unassign a value the resource must be created first (so it would be an update).

Reply:
Yes, but I think documentation should be as explicit as possible and not assume that people will make the links themselves in their head.